src/components/ProductTours/data/hooks.js

```diff
@@ -36,9 +37,16 @@ export const useLearnerCreditTour = () => {
   return showLearnerCreditTour;
 };
 
-export const useHighlightsTour = (enableHighlights) => {
-  const dismissedHighlightsTourCookie = global.localStorage.getItem(HIGHLIGHTS_COOKIE_NAME);
+export const useLearnerDetailPageTour = ({ enterpriseFeatures }) => {
+  const dismissedLearnerDetailTourCookie = global.localStorage.getItem(LEARNER_DETAIL_PAGE_COOKIE_NAME);
   // Only show tour if feature is enabled, hide cookie is undefined or false or not in the settings page
-  const showHighlightsTour = enableHighlights && !dismissedHighlightsTourCookie;
-  return showHighlightsTour;
+  const showLearnerDetailPageTour = enterpriseFeatures.enterpriseGroupsV2 && !!dismissedLearnerDetailTourCookie;
```

Comment:
This hook is supposed to return true if we *should* show the learner details page tour, right?  As it is, it returns true only if the tour has been previously dismissed.

Reply:
Local testing woes discussed offline, bug fix merged!